src/data/basic_dataset.py

```diff
@@ -75,10 +71,13 @@ def __init__(
                         image_paths["CAM_FRONT"],
                     )
 
-                # NOTE: This is a simple workaround if we do not have all files available
-                if not os.path.isfile(image_path):
-                    removed += 1
-                    continue
+                with Image.open(image_path) as image:
+                    image_width, image_height = image.size
+                    if resize_factor != 1.0:
+                        image_width = int(image_width * resize_factor)
```

Comment:
We should not be setting this every time. Assuming all images are the same size is completely fine. Still, we dont really need this (See comments below)